File: Authorization.Web/ClientApp/src/app/models/api/brief.ts
import { Caveat } from './caveat';
import { UserBrief } from './user-brief';

export interface Brief {
  id: number;
  name: string;
  foreground: string;
  background: string;
  isDeleted: boolean;

  caveats: Caveat[];
  userBriefs: UserBrief[];
}

File: Authorization.Data/Entities/Brief.cs
using System;
using System.Collections.Generic;

namespace Authorization.Data.Entities
{
    public class Brief
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Background { get; set; }
        public string Foreground { get; set; }
        public bool IsDeleted { get; set; }
        
        public List<UserBrief> UserBriefs { get; set; }
        public List<Caveat> Caveats { get; set; }
    }
}